ui-tests/jupyter_server_config.py

```diff
@@ -4,6 +4,7 @@
 c.ServerApp.disable_check_xsrf = True
 c.ServerApp.open_browser = False
 c.ServerApp.root_dir = 'jlab_root'
+c.ServerApp.preferred_dir = 'jlab_root/test'
```

Comment:
Hi @mbektas, could you give me a hand, I am not sure this is the right way/place to add this check.

I created some folders to check things `https://github.com/jupyterlab/jupyterlab/pull/10667/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R44` but it seems the config option is not being picked up

---

Pinging @fcollonval

Reply:
hi @goanpeca I am not sure if we need to create `/test` directory under `jlab_root` for UI tests. what was the reason for that? UI tests upload test files into running JupyterLab instance using Contents API and those files land in `jlab_root` which is the working directory of the jlab instance. I checked the failed UI tests, they are failing because of newly created `test` directory appearing in screenshots.

Reply:
Correct the new preferred-dir option sets a folder within the root path

So now one can do 

`jupyter lab --preferred-dir ./test/`

And the root dir would be jlab_root but the file browser would be pointing to jlab_root/test folder

This is a new option and @fcollonval suggested a UI test but it seems it is not being picked up

Reply:
Looks like `preferred_dir` is the launch directory of jlab in file browser but home is still the `root-dir`. Can you confirm?

Galata resets file browser to home directory during test suite start. https://github.com/jupyterlab/galata/blob/main/packages/galata/src/galata.ts#L2191 and https://github.com/jupyterlab/galata/blob/main/packages/galata/src/galata.ts#L1104 can give you idea.

If home directory is still pointing to `root_dir` then it would cause the issue in the failed tests.

Reply:
Ja my bet, this is probably a bad idea as it should impact only one test (targeting the `preferred_dir` feature). As this is not easily achievable, I would suggest not adding a UI tests.

Reply:
I will revert. It was good to understand better how galata works though so I appreciate the initial suggestion :)